src/mmw/bundle.sh

```diff
@@ -113,6 +118,7 @@ done
 VAGRANT_COMMAND="cd /opt/app && \
     $ENSURE_DIRS_EXIST && { \
     $COPY_IMAGES_COMMAND &
+    $COPY_SIMPLE_APP_DATA_COMMAND &
```

Comment:
Sorry if this is out of context, but I just wanted to point out that the bundle script was designed to generate output directly to `DJANGO_STATIC_ROOT` to eliminate the need to run `collectstatic`.